mixins/_icons.scss

```diff
@@ -38,29 +39,32 @@
     @if $symbol == 'arrow-up' {
         content: "\2191";
     }
-    @if $symbol == 'check' {
-        content: "\2713";
-    }
     @if $symbol == 'check-sign' {
         content: "\2611";
     }
+    @if $symbol == 'check' {
+        content: "\2713";
+    }
     @if $symbol == 'circle' {
         content: "\25cf";
     }
     @if $symbol == 'clock' {
         content: "\43";
     }
-    @if $symbol == 'close' {
-        content: "\58";
-    }
     @if $symbol == 'close-bold' {
         content: "\d7";
     }
+    @if $symbol == 'close' {
+        content: "\58";
+    }
     @if $symbol == 'cloudsave' {
         content: "\63";
     }
+    @if $symbol == 'collapse' {
+        content: "\21f1";
+    }
```

Comment:
Didn't have a symbol for this before, even though it was in the font. I've added it just in case.